dsl/src/main/java/tech/pegasys/peeps/network/Network.java

```diff
@@ -92,13 +95,13 @@
   private final List<Web3Provider> nodes;
   private final List<NetworkMember> members;
 
-  private final BesuGenesisFile genesisFile;
+  private final Map<Web3ProviderType, GenesisFile> genesisFiles;
   private final PathGenerator pathGenerator;
   private final Subnet subnet;
   private final Vertx vertx;
 
   private NetworkState state;
-  private Genesis genesis;
+  private Map<Web3ProviderType, Genesis> genesisMap = new HashMap<>();
```

Comment:
nit: having map in the name isn't nice. perhaps genesisConfigs? not sure what else to call this